modules/monitoring/application/views/scripts/show/components/comments.phtml

```diff
@@ -50,6 +50,13 @@ if (empty($object->comments) && ! $addLink) {
                 <span class="comment-time">
                     <?= $this->translate('commented') ?>
                     <?= $this->timeAgo($comment->timestamp) ?>
+                <?php if ($comment->expiration): ?>
+                    <span aria-hidden="true">&#448;</span>
```

Comment:
Why are you using the encoded version of `|` here? It's useless and already cost me too much time looking up what this code represents. Use `|` instead.

Reply:
C&P from modules/monitoring/application/views/scripts/show/components/acknowledgement.phtml